scripts/pi-hole/js/queries.js

```diff
@@ -306,49 +306,17 @@ $(function () {
       }
 
       // Check for existence of sixth column and display only if not Pi-holed
-      var replytext;
-      if (data.length > 6 && !blocked) {
-        switch (data[6]) {
-          case "0":
-            replytext = "N/A";
-            break;
-          case "1":
-            replytext = "NODATA";
-            break;
-          case "2":
-            replytext = "NXDOMAIN";
-            break;
-          case "3":
-            replytext = "CNAME";
-            break;
-          case "4":
-            replytext = "IP";
-            break;
-          case "5":
-            replytext = "DOMAIN";
-            break;
-          case "6":
-            replytext = "RRNAME";
-            break;
-          case "7":
-            replytext = "SERVFAIL";
-            break;
-          case "8":
-            replytext = "REFUSED";
-            break;
-          case "9":
-            replytext = "NOTIMP";
-            break;
-          case "10":
-            replytext = "upstream error";
-            break;
-          default:
-            replytext = "? (" + parseInt(data[6]) + ")";
-        }
+      var replytext,
+        replyid = parseInt(data[5]);
```

Comment:
Can you try to specify the radix from now on? I do have #1374 but not sure when that will be tackled.

Reply:
Sure, I will look at #1374 soon as well.

Reply:
I removed `parseInt()` here as we don't need it.